experimental/addons/mysql-operator/metadata.yaml

```diff
@@ -0,0 +1,17 @@
+name: mysql-operator
+version: 2.0.4
+description: A Kubernetes operator for Mysql Cluster
+url: https://github.com/mysql/mysql-operator
+
+tags:
+  - mysql/mysql-operator
+  - 8.0.29-2.0.4
```

Comment:
we should give more userful information in tags, it's confusing by `8.0.29-2.0.4`, a magic number